src/Aliencube.AzureFunctions.Extensions.OpenApi.Core/DocumentHelper.cs

```diff
@@ -250,6 +250,30 @@ public OpenApiResponses GetOpenApiResponses(MethodInfo element, NamingStrategy n
             return schemes;
         }
 
+        /// <inheritdoc />
+        public IList<OpenApiSecurityRequirement> GetOpenApiSecurityRequirements(IDictionary<string, OpenApiSecurityScheme> securitySchemes)
```

Comment:
Let me have a look. It should be implemented sooner rather than later.

Reply:
@gsteendp Let me think about this. It's way bigger than this implementation. Can you split this PR into two parts - one for `OpenApiDocument` and the other for `OpenApiSecurityRequirement`?

Reply:
Reverted the security requirements aspect from this pull